skl2onnx/common/utils.py

```diff
@@ -4,234 +4,11 @@
 # license information.
 # --------------------------------------------------------------------------
 
-import numbers
+from onnxconverter_common.utils import *  # noqa
```

Comment:
It is not recommended to do that. The import never fails even if a function changed name.

Reply:
Will replace with function names.